software/winrm/src/main/java/org/apache/brooklyn/util/core/internal/winrm/winrm4j/Winrm4jTool.java

```diff
@@ -75,7 +78,11 @@ public Winrm4jTool(Map<String,?> config) {
     public Winrm4jTool(ConfigBag config) {
         this.bag = checkNotNull(config, "config bag");
         host = getRequiredConfig(config, PROP_HOST);
-        port = getRequiredConfig(config, PROP_PORT);
+        port = config.get(PROP_PORT.getConfigKey()) != null
```

Comment:
The best option for supporting https is to have two separate parameters 
for `port`
`winrm.port` and `winrm.https.port`. However one port to open/enable is enough for Brooklyn to work.